core/controllers/admin.py

```diff
@@ -956,14 +951,16 @@ def put(self):
                     user_id, language_code)
             elif (category ==
                   constants.CONTRIBUTION_RIGHT_CATEGORY_REVIEW_QUESTION):
-                if not user_services.can_review_question_suggestions(user_id):
+                if not user_services.can_review_question_suggestions(
+                        user_id):
                     raise self.InvalidInputException(
                         '%s does not have rights to review question.' % (
                             username))
                 user_services.remove_question_review_rights(user_id)
             elif (category ==
```

Comment:
[optiional]: Can you find a way to avoid duplicate checks in both the handler function? (not sure if that's possible)